### DISCUSSION TITLE
Is SIGAR vital?

### DISCUSSION THREAD

**Discussion by kaimikael** [inviwo/inviwo#1229] [Q&A] [answered]
I am trying to build Inviwo on macOS 12.0.1, I’ve gotten as far as to run make; the process however fails when attempting to compile the sigar package, due to the nfsstats structure no longer being defined in nfs.h.  Looking at VMWare’s site it seems sigar was end-of-lifed quite some time ago.  Is sigar important for the functioning of Inviwo, if not, how do I indicate it can be skipped in the build process?

**Comment by petersteneteg** [comment, marked as answer]
Hi, no, it is not vital. If you set `IVW_USE_SIGAR` to off in cmake, tt should still work fine. It is only used to find system information. 
And yes, it seems apple dropped that private struct in the last os release. I should probably turn off sugar by default on Mac. But have not gotten to that yet.
